docs/framework/wcf/servicemodelreg-exe.md

```diff
@@ -49,13 +49,13 @@ File name: 'System.ServiceModel, Version=2.0.0.0, Culture=neutral, PublicKeyToke
   
  ServiceModelReg.exe cannot clean up prior version entries, nor can it register the new version's entries. The only workaround is to manually edit machine.config. You can locate this file at the following location.  
   
-```  
+```console  
 %windir%\Microsoft.NET\Framework\v2.0.50727\config\machine.config   
```

Comment:
why is this pointing to the .NET framework 2.0?

Reply:
content probably hasn't been updated since then. WCF doesn't have dedicated writer resources. Ping me offline if you'd like to understand this better.